hivemind/modules/api/tests/test_signals.py

```diff
@@ -31,3 +34,35 @@ def test_user_creates_profile(self):
         self.assertEqual(profile.first_name, user.first_name)
         self.assertEqual(profile.last_name, user.last_name)
         self.assertEqual(profile.email, user.email)
+
+    def test_event_clean(self):
```

Comment:
Hey 😄 Thank you very much for the contribution first of all. Also, could you move this test into a TestCase of its own preferably the Event Model Tests as it doesn't truly fit in as a Signal. Other than that, thank you for contributing hope to see more Contributions 👍